src/index.ts

```diff
@@ -112,6 +111,12 @@ async function createWebdriverIO(opts: ProgramOpts) {
     )
 
     await runProgram('npx', ['wdio', 'config', ...(useYarn ? ['--yarn'] : []), ...(opts.yes ? ['--yes'] : [])])
+    
+    console.log('Adding scripts to package.json')
+    const isUsingTypescript = fs.existsSync('wdio.conf.ts');
+    pkgJson = require(pkgJsonPath)
+    pkgJson.scripts['wdio'] = `wdio run wdio.conf.${isUsingTypescript ? 'ts' : 'js'}`
```

Comment:
We already write this to the package.json a bit above, I believe we could just move the code down, what do you think?

Reply:
That's what I did at first but moving the whole `package.json` manipulation down would results in the following:
1) `@wdio/cli` would be installed -> a `package.json` would be created with a `devDependencies` inside
2) `runProgram(npx run ...)` would run -> which will adds more stuff inside the `package.json` file
3) at that point we won't be able to know if the actual `package.json` was freshly created by `wdio` or not, forcing us to check if each property (name, version, private) exists prior to set them (otherwise we would override the existing ones). And adding the scripts here would make it appear at the end of the file (which feels a bit odd to me as usually scripts appears at the top)

Reply:
Maybe it would make sense to just run `npn init --yes` first if no package.json is found. The `name`, `version` and `private` property is just basic minimal boilerplate and doesn't need to be set.

Reply:
I'm not sure that `npm init --yes` would be better as the result would be even more useless :
```json
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}
```
If `name`, `version` and `private` are not necessary then I would just remove the whole `package.json` thing ([#L88-L105](https://github.com/Badisi/wdio/blob/fix-14/src/index.ts#L88-L105)). The install method [#L106](https://github.com/Badisi/wdio/blob/fix-14/src/index.ts#L106) would create a package.json if not found, otherwise use the existing one, and the scripts would be added right after the wdio installation (like in the PR).

What do you think ?

Reply:
> The install method [#L106](https://github.com/Badisi/wdio/blob/fix-14/src/index.ts#L106) would create a package.json if not found

How does that package.json would look like?

Reply:
After the install method:
```json
{
  "devDependencies": {
    "@wdio/cli": "^7.16.3"
  }
}
```
After the whole process:
```json
{
    "devDependencies": {
        "@wdio/cli": "^7.16.3",
        "@wdio/jasmine-framework": "^7.16.3",
        "@wdio/local-runner": "^7.16.3",
        "@wdio/spec-reporter": "^7.16.3",
        "chromedriver": "^95.0.0",
        "ts-node": "^10.4.0",
        "typescript": "^4.4.4",
        "wdio-chromedriver-service": "^7.2.2"
    },
    "scripts": {
        "wdio": "wdio run wdio.conf.ts"
    }
}
```
Scripts is at the bottom as we only adds it at the end, but I don't know if it's a real issue.

Reply:
this seems fine to me